keras_retinanet/models/retinanet.py

```diff
@@ -103,14 +103,15 @@ def default_regression_model(num_anchors, pyramid_feature_size=256, regression_f
 
 def __create_pyramid_features(C3, C4, C5, feature_size=256):
     # upsample C5 to get P5 from the FPN paper
-    P5           = keras.layers.Conv2D(feature_size, kernel_size=1, strides=1, padding='same', name='P5')(C5)
+    P5           = keras.layers.Conv2D(feature_size, kernel_size=1, strides=1, padding='same', name='C5_reduced')(C5)
```

Comment:
Verified this is in line with Faster RCNN paper and their Pyramid outline, including the exceptions noted in the Focal Loss paper. This was done at the time @hgaiser was writing.